src/ol_infrastructure/infrastructure/aws/eks/__main__.py

```diff
@@ -62,6 +62,8 @@
     "EXTERNAL_DNS_CHART": "1.15.0",
     "TRAEFIK_CHART": "v31.0.0",
     "VAULT_SECRETS_OPERATOR_CHART": "0.8.1",
+    "VAULT_SECRETS_OPERATOR": "0.8.1",
+    "OPEN_METADATA": "1.5.5",
```

Comment:
Should this be coming from `versions.py`?

Reply:
mmm, slipped in when I was rebasing, thanks!